### PR TITLE
AC Automated Test - LRAC-8494 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRAC-8494

### CODE REVIEW

**Comment by epark18** [comment]
Not sure if this one requires making a separate refactor ticket.

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCompositionOverview.testcase [hunk 1]
```diff
@@ -30,12 +30,45 @@ definition {
...
+			ACNavigation.goToSegments();
+
+			ACUtils.searchBar(searchItem = "Dynamic Segment Test");
+
+			ACSegments.accessSegment(segmentName = "Dynamic Segment Test");
+		}
+
+		task ("Assert the Include Anonymous Label is present") {
+			ACSegments.viewIncludeAnonymousLabel();
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCompositionOverview.testcase:71]: Hi @epark18, I think you could also check if the segment overview shows the count of some anonymous user, then you should connect the AC with the DXP and make a login with an anonymous user and check if the segment shows the count of that user and the label like you did

**Pull request closed without merging**